snippets/csharp/how-to/json/Program.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace ReaderSample
+{
+    class Program
+    {
+        private static readonly byte[] s_nameUtf8 = Encoding.UTF8.GetBytes("name");
+        private static readonly byte[] s_universityOfUtf8 = Encoding.UTF8.GetBytes("University of");
+
+        public static void Main(string[] args)
```

Comment:
You can use `public static async Task Main` since this relies on .NET Core 3. That would enable `await` expressions below instead of the `GetAwaiter().GetResult()` syntax.

Reply:
I was trying to do that, especially since @richlander provided the feedback that I should avoid using Task.Run/.Wait/.Result pattern. Just returning a regular task (using await) works as expected. Ty for the feedback!